src/CMake/nativeTests.cmake

```diff
@@ -5,28 +5,35 @@
 # Custom variables imported by this CMake stub which should be defined by parent CMake:
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`